app/views/articles/new.html.erb

```diff
@@ -1,29 +1,35 @@
-<h1>New Article</h1>
+<tr style="height:10px"></tr>
+<tr>
+  <td>
+    <h1>New Article</h1>
 
-<%= form_with model: @article do |form| %>
-  <div>
-    <div>
-      <%= form.label :title %>
-    </div>
-    <%= form.text_field :title %>
-    <% @article.errors.full_messages_for(:title).each do |message| %>
-      <div><%= message %></div>
-    <% end %>
-  </div>
+    <%= form_with model: @article do |form| %>
```

Comment:
I would turn the `article` method you already have in the controller into a helper method and use that here instead since it guarantees that it will be set.